helpers.js

```diff
@@ -60,4 +60,20 @@ module.exports = {
     const time = new Date().getHours();
     return time === 6 || time === 14 || time === 22
   },
-};
+
+  // Every hour check if the current time is within the ranges
+  setInterval: () => {
```

Comment:
This should not be in helpers, this is an invoker and should be with the event listeners. Also does this syntax even work? confused